frontend/automation/cypress/support/step_definitions/wallets.step.js

```diff
@@ -8,7 +8,8 @@ const walletPage = new WalletPage()
 const keyPairPage = new KeypairPage()
 
 Given('I am on the wallets page', () => {
-  cy.visit('#/wallet')
+  cy.visit('#/wallet', { timeout: 3000 })
```

Comment:
question (non-blocking): why extended timeout? I assume load takes a while, if so could we add a comment to that effect

Reply:
Not sure why I added this,removed